src/main/java/com/amihaiemil/docker/Execs.java

```diff
@@ -34,6 +37,15 @@
  */
 public interface Execs {
 
+    /**
+     * Create a exec with ID or name of container.
+     * @param id ID or name of container.
+     * @param config Exec configuration.
+     * @return Created Exec.
+     * @throws IOException If something goes wrong.
+     */
+    Exec create(final String id, final JsonObject config) throws IOException;
```

Comment:
@lumimul This method should be on the ``Container`` interface and should only work if the Container is running. According to the [documentation](https://docs.docker.com/engine/api/v1.40/#operation/ContainerExec): "Run new commands inside running containers"

Also, the path of the request is ``/containers/{id}/exec``, which is another indicator that it should be on the ``Container`` interface: path building throughout the library should be fluent, we shouldn't have to replace strings or do other kind of manipulations.

Reply:
i am not understand what this method will return
id of exec or  RtExec? if RtExec need replace uri (baseuri/container/hash -> baseuri/exec/hash)

Reply:
@lumimul This method will return the created ``Exec``, like this:

```java
final class RtContainers implements Containers {
    //...
    private final Docker docker;

    //...

    public Exec create(final JsonObject config){
        //do the HTTP call
        final String execId = ...;
        return this.docker.execs().get(execId);
    } 
    //...
}
```

Reply:
fix it